crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -725,20 +725,6 @@ where
 
         let parent_slot = extract_pre_digest(&parent_header).map(|d| d.slot())?;
 
-        // Make sure that slot number is strictly increasing
-        let slots_since_parent = match pre_digest.slot().checked_sub(*parent_slot) {
-            Some(slots_since_parent) => {
-                if slots_since_parent > 0 {
-                    Slot::from(slots_since_parent)
-                } else {
-                    return Err(Error::SlotMustIncrease(parent_slot, pre_digest.slot()));
-                }
-            }
-            None => {
-                return Err(Error::SlotMustIncrease(parent_slot, pre_digest.slot()));
-            }
-        };
-
         let parent_subspace_digest_items = if block_number.is_one() {
```

Comment:
Dont we still need to check if the slot is increasing ?

Reply:
Very good catch, indeed. This was combined with a the diff before, but the check itself must still be present, thanks!